main/src/cgeo/geocaching/models/Geocache.java

```diff
@@ -301,9 +301,7 @@ public boolean gatherMissingFrom(final Geocache other) {
         if (StringUtils.isBlank(getDescription())) {
             description = other.getDescription();
         }
-        // FIXME: this makes no sense to favor this over the other. 0 should not be a special case here as it is
-        // in the range of acceptable values. This is probably the case at other places (rating, votes, etc.) too.
```

Comment:
I think the FIXME still holds for rating, etc. Please move it to the rating check below and adapt the text accordingly.

Reply:
done